src/sections/pages/followers/index.js

```diff
@@ -0,0 +1,152 @@
+            <PageHeader
+                className="react-rainbow-admin-followers_header"
+                title="Followers"
+                description="Lorem ipsum dolor sit amet, consectetur adipiscing elit." />
+
+            <section className="react-rainbow-admin-followers_section rainbow-align-content_space-between rainbow-p-top_large">
+                <Tile
+                    className="react-rainbow-admin-followers_tile-facebook"
+                    title="facebook"
+                    label="143K"
+                    variant="inverse"
+                    icon={<FacebookIcon className="react-rainbow-admin-followers_tile-icon" />} />
+
+                <Tile
+                    title="google"
+                    label="20K"
+                    icon={<GoogleIcon className="react-rainbow-admin-followers_tile-icon" />} />
+
+                <Tile
+                    className="react-rainbow-admin-followers_tile-twitter"
+                    title="twitter"
+                    label="13K"
+                    variant="inverse"
+                    icon={<TwitterIcon className="react-rainbow-admin-followers_tile-icon" />} />
+
+                <Tile
+                    className="react-rainbow-admin-followers_tile-linkedin"
+                    title="linkedin"
+                    label="10K"
+                    variant="inverse"
+                    icon={<LinkedinIcon className="react-rainbow-admin-followers_tile-icon" />} />
+            </section>
+
+            <section className="react-rainbow-admin-followers_section rainbow-align-content_space-between rainbow-p-top_large">
+                <Card className="react-rainbow-admin-followers_card rainbow-p-vertical_medium rainbow-p-horizontal_small">
+                    <div className="rainbow-flex rainbow-justify_spread">
+                        <h1 className="react-rainbow-admin-followers_card-chart--title rainbow-color_dark-1 rainbow-p-bottom_medium">facebook users</h1>
+                        <Badge className="react-rainbow-admin-followers_badge--success">
+                            <FontAwesomeIcon icon={faArrowUp} pull="left" />
+                            20.5%
+                        </Badge>
+                    </div>
+                    <p className="react-rainbow-admin-followers_chart-title rainbow-align-content_center rainbow-font-size-text_x-small rainbow-color_gray-3">
+                        Total users
+                    </p>
+                    <Chart
+                        className="react-rainbow-admin-followers_chart rainbow-p-top_x-small"
+                        maintainAspectRatio={false}
+                        labels={['January', 'February', 'March', 'April', 'May', 'Jun', 'Jul']}
+                        type="line"
+                        showLegend={false}>
+                        <Dataset values={[270, 190, 350, 930, 500, 960, 1000]} borderColor="#3c5997" />
+                    </Chart>
+                </Card>
```

Comment:
I think this is very similar to one in the dashboard, let's try to create a component for this